test/e2e/antreapolicy_test.go

```diff
@@ -37,13 +37,13 @@ import (
 
 // common for all tests.
 var (
-	allPods          []Pod
-	k8sUtils         *KubernetesUtils
-	allTestList      []*TestCase
-	pods, namespaces []string
-	podIPs           map[string]string
-	p80, p81, p8080  int
-	p8085            int32
+	allPods                []Pod
+	k8sUtils               *KubernetesUtils
+	allTestList            []*TestCase
+	pods, namespaces       []string
+	podIPs                 map[string]string
+	p80, p81, p8080, p8082 int
+	p8081, p8085           int32
```

Comment:
Orthogonal to this PR but looking at the code it seems that they could all be int32s?

Reply:
this comment makes sense.. and i started updating it and ended up in a cascade of changes .. will do it in a separate PR

Reply:
no i did not forget https://github.com/vmware-tanzu/antrea/pull/1854